src/main/java/teammates/ui/template/CoursePagination.java

```diff
@@ -0,0 +1,40 @@
+package teammates.ui.template;
+
+import java.util.List;
+
+public class CoursePagination {
+    private String previousPageLink;
+    private String nextPageLink;
+    private List<String> coursePaginationList;
+    private String activeCourse = "";
```

Comment:
Is this initialisation required?
It seems to be already handled in `*PageData`.